backend/src/main/java/com/chingu/ChinguBoard/model/User.java

```diff
@@ -68,5 +83,47 @@ public void setLastName(String lastName) {
         this.lastName = lastName;
     }
 
+    public Role getRole() {
+        return this.role;
+    }
+
+    public void setRole(Role role) {
+        this.role = role;
+    }
+
+    @Override
```

Comment:
what does @Override do? Is there documentation you could point me towards? Thank yo.

Reply:
The User class implements the UserDetail interface. An interface in a way is an abstraction of classes that can provide methods that concrete classes will have to implement. The UserDetail interface has a method that needs to be implemented by the User class, which it does with the `@Override`  annotation